src/test/java/RegexTest.java

```diff
@@ -63,6 +63,22 @@ public void testLog() {
         testFor(regex, str);
     }
 
+    private static String repeat(String str, int count) {
+        StringBuilder result = new StringBuilder(str.length() * count);
+        for (int i = 0; i < count; i++) {
+            result.append(str);
+        }
+        return result.toString;

```

Comment:
Should it be `return result.toString()`?

Reply:
@zhztheplayer You're correct. I tested only the Java 11 version and wrote a fix for versions < 11 by hand without testing. That fix included a typo which you correctly noted. I fixed that typo.